config/v1/types_console.go

```diff
@@ -35,3 +38,10 @@ type ConsoleList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Console `json:"items"`
 }
+
+type ConsoleAuthentication struct {
+	// An optional, absolute URL to redirect web browsers to after logging out of the
+	// web console. If not specified, it will redirect to the default login page.
+	// +optional
+	LogoutRedirect string `json:"logoutRedirect,omitempty"`
```

Comment:
@enj sufficient comment?  Does this handle your previous objection? If no, any suggestion so we can get this merged? Thanks!

Reply:
resolving this.